### PR TITLE
File reorganization and some cleanup

### PR DESCRIPTION
**Pull request by Shtoyan**
- Added proper README.
- Added detailed build info.
- Moved all configs to Configs folder.
- Moved all other files to their proper folders inside Docs directory.
- Fixed CHANGELOG's icon.
- MD Lint'd CHANGELOG.
- Added player steam acc links to CHANGELOG.

### CODE REVIEW

**Comment by poosh** [comment]
- Rename Docs/Logo to Docs/img (in case other images will be added later)
- Docs/Redundant can be deleted - those files are not needed anymore, and those are saved in git log history anyway.
- *.cmd scripts need to be moved back to the root folder - otherwise, relative paths get broken.
- ~~Can links to _local_ resources in .md files be replaced with relative URL? For example, "Logo/ScrNBalanceLogo256.jpg" or "../readme.md".~~ (nevermind, already done)

File: README.md [hunk 1]
```diff
@@ -1,3837 +1,81 @@
...
+  * Animations\ScrnAnims.ukx
+  * Sounds\ScrnSnd.uax
+  * StaticMeshes\ScrnSM.usx
+  * Textures\ScrnAch_T.utx
+  * Textures\ScrnTex.utx
+  * Textures\TSC_T.utx
+
+**EditPackages**:
+
+```cpp
```
Comment (poosh) [inline README.md:63]: `ini`, not `cpp`
```suggestion
```ini
```

File: README.md [hunk 2]
```diff
@@ -1,3837 +1,81 @@
...
+EditPackages=ServerPerks
+EditPackages=ServerPerksMut
```
Comment (poosh) [inline README.md:65]: ScrnBalance is not linked to ServerPerksMut and ScrnSP. It requires only ServerPerks, ScrnShared, and ScrnVotingHandler.
Here is a package dependency diagram: https://drive.google.com/file/d/1T8aDY5Qem8SjLX8nAImBLI2S9B0daakT/view?usp=sharing

**Comment by poosh** [comment]
And big thanks for making a proper readme!

**Comment by Shtoyan** [comment]
I think that's all if you didn't forget anything else.

**Review by poosh — APPROVED** [review]
Looks good!

**Pull request merged**